js/kucoin.js

```diff
@@ -2703,6 +2705,78 @@ module.exports = class kucoin extends Exchange {
         return this.safeInteger (config, 'cost', 1);
     }
 
+    async fetchBorrowRateHistory (code, since = undefined, limit = undefined, params = {}) {
+        /**
+         * @method
+         * @name kucoin#fetchBorrowRateHistory
+         * @description retrieves a history of a currencies borrow interest rate at specific time slots
+         * @see https://docs.kucoin.com/#margin-trade-data
+         * @param {str} code unified currency code
+         * @param {int|undefined} since timestamp for the earliest borrow rate
+         * @param {int|undefined} limit the maximum number of [borrow rate structures]
+         * @param {dict} params extra parameters specific to the kucoin api endpoint
```

Comment:
@Dan-krm For the future, let's apply the new docstring types, as noted here: https://github.com/ccxt/ccxt/pull/14199